### PR TITLE
Feature/Add X close to feedback dialog.

### PR DESCRIPTION
**Pull request by PhotoNomad0**
# Describe what your pull request addresses

- [ ] Add X close to feedback dialog

## Test Instructions
- can test with https://deploy-preview-144--gateway-edit.netlify.app/
- [ ] open app.  Click on left menu, select "bug report or feedback".
- [ ] click on X to close button in upper right:

Screen Shot 2021-04-01 at 2 49 49 PM

- [ ] Feedback page should disappear and you should return to previous page.

### CODE REVIEW

File: pages/feedback.js [hunk 1]
```diff
@@ -93,17 +102,24 @@ const SettingsPage = () => {
...
-            <h3 className='text-xl text-gray-600 font-semibold mx-8 mb-0'>
-              Submit a Bug Report or Feedback
-            </h3>
+            <div  className='flex flex-row'>
+              <h3 className='flex-auto text-xl text-gray-600 font-semibold mx-8 mb-0'>
+                Submit a Bug Report or Feedback
+              </h3>
+              <CloseIcon
+                id='settings_card_close'
+                className={`${classes.pointerIcon} flex-none mt-4 mr-7 mb-0`}
```
Comment (mannycolon) [inline pages/feedback.js:119]: Great job using Tailwindcss, should we use the `cursor-pointer` class instead to make it consistent with the rest of the classes?
https://tailwindcss.com/docs/cursor#pointer

File: pages/feedback.js [hunk 2]
```diff
@@ -48,21 +49,29 @@ const useStyles = makeStyles(theme => ({
...
+  pointerIcon: {
+    cursor: 'pointer',
+  },
 }))
 
 const SettingsPage = () => {
   const classes = useStyles()
+  const router = useRouter()
   const categories = ['Bug Report', 'Feedback']
-  const [sumitting, setSumitting] = useState(false)
```
Comment (mannycolon) [inline pages/feedback.js:56]: Thanks for fixing the typos 🤦

Comment (PhotoNomad0) [reply pages/feedback.js:56]: No Problem, I'm sure you would help me out also.

**Review by mannycolon — APPROVED** [review]
Good job!

**Pull request merged**